cfbs/commands.py

```diff
@@ -289,7 +289,7 @@ def init_command(index_path=None, non_interactive=False) -> int:
             os.unlink(cfbs_filename())
             return 1
 
-    print("Initialized")
+    print("Initialized an empty project called '{}' in {}".format(name, cfbs_filename()))
```

Comment:
It can be whatever you want it to be 😅 The reason was probably that the name is user input, can include spaces, which the filename is just `cfbs.json`. I can add the quotes 🚀

Reply:
Done.